.dockerignore

```diff
@@ -1,17 +1,32 @@
 node_modules
+**/__pycache__
+*.pyc
+
 Dockerfile*
 docker-compose*
 .dockerignore
 .git
+.github/
 .vscode
+deployment/
+docs/
+docker/
 env/
 local.py
 /doc/_build/
+/nlx
 
 /htmlcov
 .coverage
+# tests and test assets
+src/**/tests/**/*.py
+src/**/tests/files/
```

Comment:
if we don't go for the `src/**/tests/*` then we should maybe also include `src/**/tests/responses/` `src/**/tests/exports/` ?